### PR TITLE
design user view

### PR DESCRIPTION
**Pull request by kienpham9819**
## Related Tickets:
- [#29911](https://edu-redmine.sun-asterisk.vn/issues/29911)

## WHAT this PR does?
- design user view

## Notes Impacted Areas:
- N/A.

## Attachments:
- Screencast-from-03-10-2020-17_14_55

### CODE REVIEW

File: resources/views/users/admin/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    <li class="nav-item">
+                        <a class="nav-link disabled text-dark font-weight-bold h5" href="#">
+                            <i class="fas fa-school"></i>
+                            {{ trans('course.management') }}
+                        </a>
+                        <ul class="nav flex-column ml-4">
+                            <li class="nav-item">
+                                <a class="nav-link text-dark" href="#">
+                                    <i class="fas fa-caret-right"></i>
+                                    course 1
```
Comment (TDM1911) [inline resources/views/users/admin/index.blade.php]: Dùng @foreach () hiển thị các course.

File: resources/views/users/admin/list.blade.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                    {{ trans('user.add_user_form') }}
+    </a>
+    <div class="collapse @if ($errors->any()) show @endif" id="collapseExample">
+        <div class="card card-body">
+            <form action="{{ route('users.store') }}" method="POST" role="form">
+                @csrf
+                <div class="form-group">
+                    <label for="name_user">{{ trans('user.name') }}</label>
+                    @error('name')
+                    <span class="text-danger">{{ $message }}</span>
```
Comment (TDM1911) [inline resources/views/users/admin/list.blade.php]: 4 space
```suggestion
                        <span class="text-danger">{{ $message }}</span>
```

File: resources/views/users/admin/list.blade.php [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                    @enderror
+                    <input type="text" class="form-control" name="name" id="name_user" placeholder="{{ trans('user.type_name') }}" value="{{ old('name') }}">
+                </div>
+                <div class="form-group">
+                    <label for="email_user">{{ trans('user.email') }}</label>
+                    @error('email')
+                    <span class="text-danger">{{ $message }}</span>
```
Comment (TDM1911) [inline resources/views/users/admin/list.blade.php]: 4 space

File: resources/views/users/admin/list.blade.php [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+                    @enderror
+                    <input type="email" class="form-control" name="email" id="email_user" placeholder="{{ trans('user.type_email') }}" value="{{ old('email') }}">
+                </div>
+                <div class="form-group">
+                    <label for="password_user">{{ trans('user.password') }}</label>
+                    @error('password')
+                    <span class="text-danger">{{ $message }}</span>
```
Comment (TDM1911) [inline resources/views/users/admin/list.blade.php]: 4 space

File: resources/views/users/admin/list.blade.php [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+                    @enderror
+                    <input type="password" class="form-control" name="password" id="password_user" placeholder="{{ trans('user.type_password') }}" value="{{ old('password') }}">
+                </div>
+                <div class="form-group">
+                    <label for="repassword">{{ trans('user.password_confirm') }}</label>
+                    @error('password_confirm')
+                    <span class="text-danger">{{ $message }}</span>
```
Comment (TDM1911) [inline resources/views/users/admin/list.blade.php]: 4 space

File: resources/views/users/admin/restore.blade.php [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+                    {{ trans('user.add_user_form') }}
+    </a>
+    <div class="collapse @if ($errors->any()) show @endif" id="collapseExample">
+        <div class="card card-body">
+            <form action="{{ route('users.store') }}" method="POST" role="form">
+                @csrf
+                <div class="form-group">
+                    <label for="name_user">{{ trans('user.name') }}</label>
+                    @error('name')
+                    <span class="text-danger">{{ $message }}</span>
```
Comment (TDM1911) [inline resources/views/users/admin/restore.blade.php]: 4 space

File: resources/views/users/admin/restore.blade.php [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+                    @enderror
+                    <input type="text" class="form-control" name="name" id="name_user" placeholder="{{ trans('user.type_name') }}" value="{{ old('name') }}">
+                </div>
+                <div class="form-group">
+                    <label for="email_user">{{ trans('user.email') }}</label>
+                    @error('email')
+                    <span class="text-danger">{{ $message }}</span>
```
Comment (TDM1911) [inline resources/views/users/admin/restore.blade.php]: 4 space

File: resources/views/users/admin/restore.blade.php [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+                    @enderror
+                    <input type="email" class="form-control" name="email" id="email_user" placeholder="{{ trans('user.type_email') }}" value="{{ old('email') }}">
+                </div>
+                <div class="form-group">
+                    <label for="password_user">{{ trans('user.password') }}</label>
+                    @error('password')
+                    <span class="text-danger">{{ $message }}</span>
```
Comment (TDM1911) [inline resources/views/users/admin/restore.blade.php]: 4 space

File: resources/views/users/admin/restore.blade.php [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+                    @enderror
+                    <input type="password" class="form-control" name="password" id="password_user" placeholder="{{ trans('user.type_password') }}" value="{{ old('password') }}">
+                </div>
+                <div class="form-group">
+                    <label for="repassword">{{ trans('user.password_confirm') }}</label>
+                    @error('password_confirm')
+                    <span class="text-danger">{{ $message }}</span>
```
Comment (TDM1911) [inline resources/views/users/admin/restore.blade.php]: 4 space

File: resources/views/users/admin/restore.blade.php [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+                                @csrf
+                                @method('delete')
+                                <button type="submit" class="border-0 bg-white">
+                                    <i class="fas fa-trash text-danger"></i>
+                                </button>
+                            </form>
+                        </td>
+                    </tr>
+                @empty
+                <p>Empty</p>
```
Comment (TDM1911) [inline resources/views/users/admin/restore.blade.php]: - Indent 4 space.
- Localization từ empty

File: resources/lang/en/user.php [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+    'disable' => 'Disable',
+    'edit' => 'Edit user',
+    'edit_title' => 'User edit',
+    'email' => 'Email',
+    'excel' => 'Import excel',
+    'force_delete' => 'Force Delete',
+    'import' => 'Import excel',
+    'name' => 'Name',
+    'new_password' => 'Enter new password',
+    'noti_add' => 'You added success',
```
Comment (TDM1911) [inline resources/lang/en/user.php]:
```suggestion
    'noti_add' => 'New user added',
```

File: resources/lang/en/user.php [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+    'noti_delete' => 'You deleted success',
```
Comment (TDM1911) [inline resources/lang/en/user.php]:
```suggestion
    'noti_delete' => 'User deleted',
```

File: resources/views/users/admin/edit.blade.php [hunk 13]
```diff
@@ -0,0 +1,113 @@
+@extends('users.admin.index')
+
+@section('breadcrumb')
+    <li class="breadcrumb-item text-capitalize" aria-current="page">
+       <a href="{{ route('users.index') }}">{{ trans('user.title_list') }}</a>
+    </li>
+    <li class="breadcrumb-item active text-capitalize" aria-current="page">
+        {{ trans('Edit') . '-' . $user->name}}
```
Comment (TDM1911) [inline resources/views/users/admin/edit.blade.php]:
```suggestion
        {{ trans('Edit') . '-' . $user->name }}
```

File: resources/views/users/admin/list.blade.php [hunk 14]
```diff
@@ -0,0 +1,162 @@
...
+                            <thead class="thead-dark">
+                                <tr>
+                                    <th>#</th>
+                                    <th>{{ trans('role.name') }}</th>
+                                    <th>{{ trans('role.option') }}</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                @isset($roles)
+                                    @foreach($roles as $key => $role)
```
Comment (TDM1911) [inline resources/views/users/admin/list.blade.php]:
```suggestion
                                    @foreach ($roles as $key => $role)
```

File: resources/views/users/admin/restore.blade.php [hunk 15]
```diff
@@ -0,0 +1,118 @@
+@extends('users.admin.index')
+
+@section('breadcrumb')
+    <li class="breadcrumb-item active text-capitalize" aria-current="page">
+       {{ trans('user.title_list') }}
+    </li>
+@endsection
+
+@section('admin')
+    @if(session('message'))
```
Comment (TDM1911) [inline resources/views/users/admin/restore.blade.php]:
```suggestion
    @if (session('message'))
```

**Comment by kienpham9819** [comment]
ready

File: resources/views/users/admin/edit.blade.php [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+                    <thead class="thead-dark">
+                        <tr>
+                            <th>#</th>
+                            <th>{{ trans('role.name') }}</th>
+                            <th>{{ trans('role.option') }}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        @isset($roles)
+                            @foreach($roles as $key => $role)
```
Comment (huyenttm-0901) [inline resources/views/users/admin/edit.blade.php]:
```suggestion
                        @isset ($roles)
                            @foreach ($roles as $key => $role)
```

**Comment by kienpham9819** [comment]
ready

File: resources/views/users/admin/list.blade.php [hunk 17]
```diff
@@ -0,0 +1,162 @@
...
+    @if (session('message'))
+        <div class="alert alert-success">
+            <button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
+            <strong>{{ session('message') }}</strong>
+        </div>
+    @endif
+    <a class="btn btn-primary text-capitalize" data-toggle="collapse" href="#collapseExample" role="button" aria-expanded="false" aria-controls="collapseExample">
+                    <i class="fas fa-user-plus"></i>
+                    {{ trans('user.add_user_form') }}
+    </a>
```
Comment (huyenttm-0901) [inline resources/views/users/admin/list.blade.php]: indent 4 spaces

**Comment by kienpham9819** [comment]
ready

**Pull request merged**